docs/PUML-mesh-format.rst

```diff
@@ -77,7 +95,7 @@ Other boundary formats (``int64``) have a 16-bit offset and use 0xffff as a mask
 | 3: dynamic rupture
 | 4: dirichlet
 | 5: absorbing
-| 6: periodic
+| 6: periodic (behaves similarly to 0)
```

Comment:
then maybe consider:

identified (used for topological face mapping, e.g. periodic or stitched domains)